backend/model/jobWorker.js

```diff
@@ -453,23 +456,16 @@ class JobWorker extends Job {
 
       console.log("pythonPathname:", pythonPathname);
 
-      //Convert .ipynb => .py
-      if (jupyterNotebookPathname) {
-        await exec("jupyter nbconvert --to script", jupyterNotebookPathname);
-      }
-
-      const trainedModelFileName = await installNotebookDependencies(
-        pythonPathname
-      );
+      await installNotebookDependencies(pythonPathname);
 
       await updateNotebookMorphwareTerms(
         pythonPathname,
         this.downloadPath + "/"
       );
 
-      await exec("python3", pythonPathname, trainingDataPathname);
+      await exec("python3", pythonPathname, "morphware_train");
```

Comment:
NIT: We can store "morphware_train" and "morphware_validate" as an enum somewhere and just reference that. Instead of having to type it out in worker and validator